src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -63,4 +65,15 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <FscToolPath Condition=" '$(FscToolPath)' == '' ">$(MSBuildThisFileDirectory)</FscToolPath>
     <FscToolExe Condition="'$(OS)' == 'Unix' and '$(FscToolExe)' == ''">RunFsc.sh</FscToolExe>
   </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="$(DefaultValueTuplePackageVersion)" >
+      <DefaultInclude>true</DefaultInclude>
+    </PackageReference>
+
+    <PackageReference Include="FSharp.Core" Version="$(DefaultFSharpPackageVersion)">
```

Comment:
is possibile to add here a condition to NOT create these?
like https://github.com/dotnet/sdk/blob/feca75bd43f6374715621024d930164de8bde67c/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props#L48 does

so same `Condition=" '$(DisableImplicitFSharpCoreReference)' != 'true' "` ?

i mean, why renamed from `DisableImplicitFSharpCoreReference` to `DefaultValueTuplePackageVersion`?
it will make a special case in other tools and program (paket but also customized existing fsproj)

Reply:
@enricosada 

the order that properties are set is:
1.     The props file
2.     The user project file
3.     The targets file

So if we want there to be an implicit reference, that the developer can amend by setting DisableImplicitFSharpCoreReference.  We need to add it in props and remove it in targets.

The existing parameters DisableImplicitFSharpCoreReference and FSharpCoreImplicitPackageVersion should still work fine, unless I messed up, which I may have.  The DefaultValueTuplePackageVersion and DefaultFSharpCorePackageVersion are just a way of parameterizing the 4.2.*.

So the complexity of add then remove, are primarily to support the old Disable and version setting mechanism.

I hope this clears some stuff up.